src/lib/crypto/kimchi_bindings/stubs/dune

```diff
@@ -14,19 +14,29 @@
     (run cargo_target_dir/cargo_target_dir.exe)))))
 
 (rule
+ (enabled_if (= %{env:MARLIN_PLONK_STUBS=n} n))
  (targets libwires_15_stubs.a)
  (deps
   Cargo.toml
   (source_tree src)
+  (source_tree binding_generation)
   (source_tree ../../proof-systems))
  (locks /cargo-lock)
  (action
   (progn
+   (setenv
+    CARGO_TARGET_DIR
+    "./target"
```

Comment:
This change has wasted literally hours of my day today. My fault for not catching it in review, but @balsoft heads up that I'll be backing this part of the change out ASAP.

Reply:
Could you clarify what exactly is wrong? I would prefer to fix this rather than just revert the entire thing.